.travis.yml

```diff
@@ -17,7 +20,7 @@ addons:
     - lapack
     - openblas
 
-matrix:
+jobs:
```

Comment:
I switched to 'jobs' and added 'os' and 'dist' default values to silence some warnings in Travis's yaml validator.